vignettes/epiparameter.Rmd

```diff
@@ -0,0 +1,203 @@
+
+The {epiparameter} R package is a library of epidemiological parameters, with classes to handle this data and a set of functions to manipulate and use epidemiological parameters and distributions. The package also contains functionality for converting and extracting distribution parameters from summary statistics.
+
+::: {.alert .alert-primary}
+### Use case 
+
+An outbreak of a known or potentially novel pathogen is detected and key parameters such as delay distributions (e.g. incubation period or serial interval) are required to interpret early data. 
+
+{epiparameter} can provide these distributions from a selection of published sources, such as past analysis of the same or similar pathogen, in order to provide relevant epidemiological parameters for new analysis.
+:::
+
+This vignette will provide a introduction to the data stored within {epiparameter}, how to read it into R, manipulate
+the data, and the functions (and methods) implemented in the package to facilitate easy application of parameters into epidemiological pipelines.
+
+The {distributional} package is loaded as some of the {epiparameter} methods use S3 generics from this package.
+
+```{r setup}
+library(epiparameter)
+```
+
+## Working with {epiparameter} data
+
+{epiparameter} introduces two new classes for working with epidemiological parameters in R:
+
+- `<epidist>`: Contains the name of the disease, the name of the epidemiological distribution, parameters (if available) and citation information of parameter source, as well as other information.
+- `<vb_epidist>`: A list of two `<epidist>` objects for a vector-borne disease. One for the human (intrinsic) distribution, and one for the vector (extrinsic).
+
+### Library of epidemiological parameters
+
+First, we will introduce the library, or database, of epidemiological parameters available from {epiparameter}. The library is stored in the package as a JSON file and can be read into R using the `epidist_db()` function. By default all entries in the library are supplied.
+
+```{r read-in-library}
+epi_dist_db <- epidist_db()
+epi_dist_db
+```
+
+The output is a list of `<epidist>` objects, where each element in the list corresponds to an entry in the parameter database. *Technical note*: the reason the object does not look like a regular R list is because it uses a custom print method which will print a summary of the data to the console when there are more than 5 entries to prevent a large list flooding the console. When the number of database entries returned is less than or equal to 5 the print will look like a standard R list.
+
```

Comment:
The deleted text is a distraction to a user who doesn't care what's going on in the background. Maybe move it to the Design vignette?
```suggestion
The output is a list of `<epidist>` objects, where each element in the list corresponds to an entry in the parameter database. 
```

Reply:
Happy to remove this text as you propose. There is a similar point to the technical note already in the `design_priniples.Rmd`.